.gitignore

```diff
@@ -21,3 +21,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+/target/
```

Comment:
Could you add .DS_Store to this and remove the .DS_Store files below

Reply:
Yep will do